desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -7,7 +7,6 @@
 #include "core/units.h"
 #include "core/dive.h"
 #include "desktop-widgets/command.h"
-#include "desktop-widgets/command_edit.h"
```

Comment:
Another unrelated change?

Reply:
Yup, that header is not required to build this file

Reply:
Then that's unrelated to "Do updatesalinity upon screen refresh", and should have bin a separate commit stating why its removed, or rather, update the commit where you add it so it doesn't add it at all.

Reply:
Fixed. I put the line back. I will submit that change as a separate PR.

Reply:
No, Not a separate PR, correct the commit where you added it instead.

Reply:
whatever  you want  ......

Reply:
I am not up to that level in git. I have now deleted that line. This is a change in code, therefore, as far as I can see, requiring an additional change to the present PR. If you would like me to do anything else, please send  me the git instructions to do elsewise.

Reply:
https://git-scm.com/book/en/v2/Git-Branching-Rebasing
https://help.github.com/en/github/using-git/about-git-rebase

Reply:
@willemferguson: If you want to merge two commits, first find out the commit id of the commit you want to merge into with
```
git log
```
Then
```
git rebase -i COMMIT-ID^
```
(the ^ is important, because you will be presented the commits *after* the commit you specified).
This will open an editor.
Copy the line of the commit you want to merge below the line of the commit you want to merge into. Then change the first letter of the commit you want to merge to `s` for "squash into previous commit".

Then save and exit the editor and it should be done.

I suggest doing this in a test-branch, which you can create by
```
git checkout -b branch-name
```

If this doesn't work out for you, I'm sure someone of us can do this for you before merging.

Reply:
I have now submitted a unified PR which hopefully comprises only one commit and which I hope adresses all the issues raised.

Reply:
Great.

For future reference, in every case where you end up writing a list in the commit message, you should split it in different commits instead. One commit per logical change.